akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -26,43 +29,38 @@ class AskSpec extends TypedSpec with ScalaFutures {
   val behavior: Behavior[Msg] = immutable[Msg] {
     case (_, foo: Foo) ⇒
       foo.replyTo ! "foo"
-      same
+      Actor.same
     case (_, Stop(r)) ⇒
       r ! ()
-      stopped
+      Actor.stopped
   }
 
   "Ask pattern" must {
     "must fail the future if the actor is already terminated" in {
-      val fut = for {
-        ref ← system ? TypedSpec.Create(behavior, "test1")
-        _ ← ref ? Stop
-        answer ← ref.?(Foo("bar"))(Timeout(1.second), implicitly)
-      } yield answer
-      fut.recover { case _: AskTimeoutException ⇒ "" }.futureValue should ===("")
+      val ref = spawn(behavior)
+      (ref ? Stop).futureValue
+      val answer = ref ? Foo("bar")
+      answer.recover { case _: AskTimeoutException ⇒ "ask" }.futureValue should ===("ask")
```

Comment:
We seem to use `should equal()`, `should be()`, `should ===()` and plain `===` - should we pick one style?

Reply:
Yes we should :), i'll make this file consistent